modules/python3-cryptography.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "python3-cryptography",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"cryptography\" --no-build-isolation"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/d4/85/38715448253404186029c575d559879912eb8a1c5d16ad9f25d35f7c4f4c/cryptography-3.3.2.tar.gz",
```

Comment:
python3-pyopenssl below install of more recent version of this package, but not of the other two. Is there some redundancy? Doesn't one get overridden by the other?

Reply:
There's a build error with the more recent version of the cryptography module. I changed the version in pyopenssl and will no longer need this.